iceberg/iceberg-handler/src/test/queries/positive/iceberg_major_compaction.q

```diff
@@ -0,0 +1,33 @@
+--test against vectorized LLAP execution mode
+-- SORT_QUERY_RESULTS
+-- Mask neededVirtualColumns due to non-strict order
+--! qt:replace:/(\s+neededVirtualColumns:\s)(.*)/$1#Masked#/
+
+-- set hive.llap.io.enabled=true;
+-- set hive.vectorized.execution.enabled=true;
+-- set hive.optimize.shared.work.merge.ts.schema=true;
+
+CREATE TABLE ice_orc (
+    first_name STRING, 
+    last_name STRING
+ )
+STORED BY ICEBERG STORED AS ORC 
+TBLPROPERTIES ('engine.hive.enabled'='true','format-version'='2');
```

Comment:
why this is needed ```'engine.hive.enabled'='true'```, it should be by default true & no other test does that

Reply:
Done